src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+// NOTE GATE
+  return (!a);
 };
 
 function both(a, b) {
-  // your code here
+// AND GATE
+  return (a&&b);
 };
 
 function either(a, b) {
-  // your code here
+// OR GATE
+  return (a||b);
 };
 
 function none(a, b) {
-  // your code here
+// NOR GATE
+  return !(a||b);
 };
 
 function one(a, b) {
-  // your code here
+// XOR GATE
+  return (a && !b) || (!a && b);
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a===b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+return a>b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+return a<=b;
 };
 
 function isOdd(a) {
-  // your code here
+  // NUMBERS NOT DIVISIBLE BY 2, REMAINDER CAN'T BE 0!
+  return a%2 == !0;
```

Comment:
Here add some spaces: 
``
```
return a % 2 == !0
``````

I would also suggest to use the inequality operator rather than using NOT 0.
It is always suggested to use the strict equality or strict inequality operators. As they allow us to compare two or more operands by checking the equality between the values as well as their types.
In your case this would be:

`````
return a % 2 !== 0
``````